internal/commands/uninstall.go

```diff
@@ -20,9 +20,10 @@ func uninstallCmd(dockerCli command.Cli) *cobra.Command {
 	var opts uninstallOptions
 
 	cmd := &cobra.Command{
-		Use:     "uninstall INSTALLATION_NAME [--target-context TARGET_CONTEXT] [OPTIONS]",
+		Use:     "rm INSTALLATION_NAME [--target-context TARGET_CONTEXT] [OPTIONS]",
```

Comment:
Was this meant to be pushed with the version change?

Reply:
seems I messed up my branches :'(